packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -1933,7 +1934,7 @@ Generated by [AVA](https://avajs.dev).
       Please make sure it is present in the site's top-level "package.json".␊
     ␊
       Build failed with 1 error:␊
-      packages/build/tests/core/fixtures/functions_config_3/functions/function_two.js ERROR: Could not resolve "@netlify/imaginary-module-two"␊
+      tests/core/fixtures/functions_config_3/functions/function_two.js:2:80: ERROR: Could not resolve "@netlify/imaginary-module-two"␊
```

Comment:
I think this snapshot is not correct (there is a function that simplifies stack traces and the `:2:80:` should be probably stripped away. I guess there is a regex that needs to be adapted